src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,31 +33,33 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${selectedTodo.id}`}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
             <button
-              type="button"
               className="delete"
               data-cy="modal-close"
+              type="button"
+              onClick={() => {
+                setSelectedTodo(null);
```

Comment:
You can create a separate function to handle this click

Reply:
didn't change it

Reply:
sorry...at this time - done!)